src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/LoginPage.kt

```diff
@@ -19,9 +20,10 @@ class LoginPage(
             ExpectedConditions.presenceOfElementLocated(loginFormLocator)
         )
         val loginForm = driver.findElement(loginFormLocator)
-        loginForm.findElement(By.name("os_username")).sendKeys(user.name)
-        loginForm.findElement(By.name("os_password")).sendKeys(user.password)
+
+        ElementUtils.sendKeysAndValidate(driver, By.name("os_username"), user.name)
+        ElementUtils.sendKeysAndValidate(driver, By.name("os_password"), user.password)
```

Comment:
It's a perfect place for an [extension function](https://kotlinlang.org/docs/reference/extensions.html).
It'd read:
```suggestion
        loginForm.findElement(By.name("os_password")).sendKeysAndValidate(user.password)
```

Extension functions compile to statics:
![image](https://user-images.githubusercontent.com/1194704/74032143-8d7d6380-49b3-11ea-961c-208f8435078f.png)

Upsides:
* easier on the eyes
* applicable to any `WebElement`, not only the ones from `WebDriver.findElement`

- [ ] consider extension functions

Reply:
It's a little bit easier to read and use in kotlin. Will the extension function be also accessible for java consumers? (as it's a part of API)

Reply:
Yes, they'd be available as static methods.